dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/DefaultMetadataExportService.java

```diff
@@ -269,7 +270,7 @@ public MetadataExportParams getParamsFromMap( Map<String, List<String>> paramete
             Class<? extends IdentifiableObject> klass = (Class<? extends IdentifiableObject>) schema.getKlass();
 
             // class is enabled if value = true, or fields/filter/order is present
-            if ( "true".equalsIgnoreCase( parameters.get( parameterKey ).get( 0 ) ) || (parameter.length > 1 && ("fields".equalsIgnoreCase( parameter[1] )
+            if ( isSelectedClass( parameters.get( parameterKey ) ) || ( parameter.length > 1 && ( "fields".equalsIgnoreCase( parameter[1] )
```

Comment:
Wouldn't it change the final logic when you now check the whole collection and not only the first item? If you are sure about it that it won't change it or if your change is made for a reason, then fine.

Reply:
The front-end app for 2.30 and later has a work around for job configurations in order not to show them in the export front-end. This causes that jobConfigurations=true&jobConfigurations=false (which makes no sense, there should be just one of them) is added.

Reply:
Cool. If you have justification for it then fine. Makes sense.